src/openrct2-ui/windows/TrackDesignPlace.cpp

```diff
@@ -240,6 +241,30 @@ static void window_track_place_update(rct_window* w)
             window_close(w);
 }
 
+static GameActionResult::Ptr FindValidTrackDesignPlaceHeight(CoordsXYZ& loc, uint32_t flags)
+{
+    for (int32_t i = 0; i < 7; i++)
```

Comment:
```suggestion
    for (int32_t i = 0; i < 7; i++, loc.z += 8)
```

You can use the `for` to control the increments of `z` and simplify the code at the bottom.